recipes/dlo-hic-tools/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+    name: dlo_hic
+    version: "0.0.0"
+
+source:
+    url: https://github.com/Nanguage/DLO-HiC-Tools/archive/v0.0.0.tar.gz # [unix and py3k]
+    sha256: 5caddccd25fb4f86689dc8b6e85f4eb4a3de990766ddb47670097671cb403b71 # [unix and py3k]
+
+build:
+    number: 1
```

Comment:
You don't have to increase the build number when the PR isn't merged yet; just use `number: 0` here.